### PR TITLE
Mention `PremiumV2_LRS` among other Azure disk skunames

### PR DESCRIPTION
**Pull request by mpatlasov**
Per request here: https://redhat-internal.slack.com/archives/CBQHQFU0N/p1696527496821959

### CODE REVIEW

File: modules/storage-azure-create-storage-class.adoc [hunk 1]
```diff
@@ -29,8 +29,14 @@ persistent volumes.
 .. Select `kubernetes.io/azure-disk` from the drop down list.
 
 ... Enter the storage account type. This corresponds to your Azure
-storage account SKU tier. Valid options are `Premium_LRS`, `Standard_LRS`,
+storage account SKU tier. Valid options are `Premium_LRS`, `PremiumV2_LRS`, `Standard_LRS`,
 `StandardSSD_LRS`, and `UltraSSD_LRS`.
++
+[IMPORTANT]
+====
+The skuname `PremiumV2_LRS` is not supported in all regions, and even in some supported regions, not all the availability zones are supported.
```
Comment (lpettyjo) [inline modules/storage-azure-create-storage-class.adoc:37]: Please rewrite as follows:

The skuname `PremiumV2_LRS` is not supported in all regions, and in some supported regions, not all of the availability zones are supported. For more information, see link:https://learn.microsoft.com/en-us/azure/virtual-machines/disks-deploy-premium-v2[Azure doc].

**Review by lpettyjo — COMMENTED** [review]
Otherwise, LGTM!

**Comment by lpettyjo** [comment]
@mpatlasov which versions of OCP are you trying to apply this to?

**Comment by mpatlasov** [comment]
Last push updated doc text per [Lisa's comment](https://github.com/openshift/openshift-docs/pull/66070#discussion_r1376222651)

**Pull request merged**